packages/rolldown/src/cli/index.ts

```diff
@@ -28,6 +28,9 @@ async function main() {
 }
 
 main().catch((err) => {
+  if (typeof err.stack === 'string') {
+    err.stack = err.replace(err.message, '')
```

Comment:
Oh wait, is this intended? (and why typescript not complaining?)

```suggestion
    err.stack = err.stack.replace(err.message, '')
```

Reply:
My bad！😢

Reply:
It was an oversight caused by my modification after testing, where I added the condition `if (typeof err.stack === 'string')`. Since `err` is of type `any`, no error was reported.

Reply:
Ah I see `err: any`. How about also checking instance just in case?

```js
catch((err: unknown) => {
  if (err instanceof Error && typeof err.stack === 'string')
```

Reply:
> Ah I see `err: any`. How about also checking instance just in case?

Good point!